notify/twilio.py

```diff
@@ -0,0 +1,84 @@
+import json
+import logging
+import os
+
+from twilio.rest import Client
+
+from .utils import shorten_url
+
+states = json.loads(os.environ["STATES"])
+
+def format_available_message(locations):
```

Comment:
Since this is sent via SMS, I think the twitter format (one message per location) would fit better.
(notify/twitter.py)

Reply:
Good call, done.